fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -200,13 +200,7 @@ class RestoreConfig : public KeyBackedConfig {
 		Version endVersion{ ::invalidVersion }; // not meaningful for range files
 
 		Tuple pack() const {
-			return Tuple()
-			    .append(version)
-			    .append(StringRef(fileName))
-			    .append(isRange)
-			    .append(fileSize)
-			    .append(blockSize)
-			    .append(endVersion);
+			return Tuple::makeTuple(version, fileName, (int)isRange, fileSize, blockSize, endVersion);
```

Comment:
casting `isRange` to an int seems to be correct given that it's read as an int below but it's strange it wasn't that way before

Reply:
Previously, `append` only took an int type, so it was implicitly cast to an int. Now that I have an overload that accepts a boolean, this needs to be done explicitly.